docs/lvmetrics.md

```diff
@@ -11,23 +11,34 @@ The value is the free storage space size in bytes.
 
 `lvmetrics` obtains the free storage space size by watching [`lvmd`](./lvmd.md).
 
-Finalier
---------
+Finalizer
+---------
 
 `lvmetrics` adds `topolvm.cybozu.com/node` finalizer to `Node`.
 The finalizer will be processed by [`topolvm-controller`](./topolvm-controller.md)
 
 Command-line flags
 ------------------
 
-| Name       | Default                  | Description                   |
-| ---------- | ------------------------ | ----------------------------- |
-| `nodename` |                          | `Node` resource name.         |
-| `socket`   | `/run/topolvm/lvmd.sock` | UNIX domain socket of `lvmd`. |
+| Name           | Default                  | Description                        |
+| ----------     | ------------------------ | -----------------------------      |
+| `nodename`     |                          | `Node` resource name.              |
+| `socket`       | `/run/topolvm/lvmd.sock` | UNIX domain socket of `lvmd`.      |
+| `metrics-addr` | `:8080`                  | The listen address of metrics API. |
 
 Environment variables
 ---------------------
 
 - `NODE_NAME`: `Node` resource name.
 
 If both `NODE_NAME` and `nodename` flag are given, `nodename` flag is preceded.
+
+
+Prometheus metrics
+------------------
+
+`lvmetrics` exposes Prometheus metrics at `/metrics` endpoint.
+
+| Name                           | Type                     | Description                                                                                         |
+| ----------                     | ------------------------ | -----------------------------                                                                       |
+| `lvmetrics_vg_available_bytes` | Gauge                    | available bytes of LVM VG mangaged by lvmd. This value is same with node annotation by lvmetrics |
```

Comment:
I'd like to suggest `topolvm_volumegroup_available_bytes` in place of this
because `lvmetrics` will be merged into `topolvm-node`...

Reply:
:OK: